lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: Condition
+field_name_base_url: http://hl7.org/fhir/r4/condition-definitions.html#Condition
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | inactive | resolved
+  url: http://hl7.org/fhir/r4/condition-definitions.html#Condition.clinicalStatus
+  binding:
+    description: The clinical status of the condition.
+    terminology:
+      - display: Condition Clinical Status Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-clinical
+        info_link: http://hl7.org/fhir/r4/valueset-condition-clinical.html
+
+- name: verificationStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: unconfirmed | confirmed | refuted | entered-in-error
+  url: http://hl7.org/fhir/r4/condition-definitions.html#Condition.verificationStatus
+  binding:
+    description: The verification status to support or decline the clinical status of the condition or diagnosis.
+    terminology:
+      - display: Condition Verification Status Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-ver-status
+        info_link: http://hl7.org/fhir/r4/valueset-condition-ver-status.html
+
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: problem-list-item | encounter-diagnosis
+  url: http://hl7.org/fhir/r4/condition-definitions.html#Condition.category
+  example: |
+    {
+      "category": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/condition-category",
+            "code": "problem-list-item",
+            "display": "Problem List Item"
+          }
+        ],
+        "text": "Diagnosis"
+      }
+    }
+  binding:
+    description: A category assigned to the condition.
+    note: Category code `problem-list-item`is supported for search and retrieve functionality.
```

Comment:
We don't support retrieve yet.

Reply:
Fixed. [diff](https://github.com/cerner/fhir.cerner.com/pull/273/commits/4872fd105329788054750c0c23ab3e1c4ec4459a#diff-abb99d2139d79f8a2a8f1d214f5f6882R52)